build.gradle

```diff
@@ -152,11 +152,11 @@ publishing {
         mavenLocal()
     }
 }
-
+/*
```

Comment:
I think you accidentally committed this change. I need to figure out a way to gate this signing block, but until then I need it present.